scripts/apps/authoring-react/data-layer.ts

```diff
@@ -122,6 +124,28 @@ function getArticleContentProfile<T>(item: IArticle, fieldsAdapter: IFieldsAdapt
             }
         }
 
+        // TODO: write an upgrade script and remove hardcoding
+        // after angular based authoring is removed from the codebase
+        if (item.type === 'picture') {
```

Comment:
is it for picture only or all media including audio/video?

Reply:
I mean in old authoring

Reply:
Checked, we support audio and video too.

Reply:
Add 'graphic'

Reply:
After looking at the code a bit I found that we consider 'media' type to be only `picture`, `video` and `audio` so do you think I should still add `graphic` type in the condition?
![image](https://user-images.githubusercontent.com/45574768/215089131-7237edb5-aaac-4e65-b569-a07daa6eb93e.png)